### PR TITLE
apparmor,domoticz: use $(PYTHON3_VERSION) instead of hardcoding it in the Makefile

### PR DESCRIPTION
**Pull request by cotequeiroz**
Maintainer: @oskarirauta, @dwmw2 
Compile tested: mediatek, aarch64
Run tested: none

Description:
The packages fail to build with python 3.10, because the python version was hardcoded to the packages' Makefile, and there was a recent python change from 3.9 to 3.10.

Instead of hardcoding the pyton version, use `PYTHON3_VERSION` from python3-version.mk (included by `python3-package.mk`).

I have not run-tested them, but I did check that the apparmor files where properly installed in `/usr/lib/python3.10/site-packages`.

### CODE REVIEW

**Review by oskarirauta — APPROVED** [review]
Looks good...

File: utils/domoticz/Makefile [hunk 1]
```diff
@@ -75,7 +76,7 @@ CMAKE_OPTIONS += \
 
 TARGET_CFLAGS += -flto
 TARGET_CXXFLAGS += -DWITH_GPIO -flto
-TARGET_LDFLAGS += -lpython3.9
+TARGET_LDFLAGS += -lpython$(PYTHON3_VERSION)
```
Comment (neheb) [inline utils/domoticz/Makefile:79]: Wrong. This just needs to be removed. This was a hack from before domoticz had proper python 3.9 support. It's not needed anymore.

**Comment by neheb** [comment]
The egg-info thing is weird.

@jefferyto know anything about that? I don't see other packages installing that.

File: utils/domoticz/Makefile (hunk 1 above)
Comment (cotequeiroz) [reply utils/domoticz/Makefile:79]: OK

**Comment by cotequeiroz** [comment]
Maybe @jefferyto knows their relevance, but the egg-info files or directories are quite common: 
```
src/openwrt/staging_dir/target-aarch64_cortex-a53_musl/root-mediatek $ find usr/lib/python3.9/site-packages/ -name '*egg-info' | wc -l
195
```
This was an from an older `CONFIG_ALL=y` staging dir that I had in hand, but it should not change that much with the python3.10 update.

**Comment by BKPepe** [comment]
Let's merge this. We can take a look at ```egg-info``` files later.

**Comment by jefferyto** [comment]
egg-info files are how on-device pip knows what Python packages are installed. They are usually packaged automatically by the Python package build process in python3-package.mk, so it's a little unusual to see them "manually" installed like here.

**Pull request merged**